### ISSUE TITLE
Missing block for motor on normal MakerBit at P12

### ISSUE THREAD

**Issue by MKleinSB** [1010Technologies/pxt-makerbit-motor#1]
Ich wollte den Motor am Motorblock vom normalen MakerBit (P12) testen, für den gibt es aber keinen Block. Man kann natürlich selbst den digitalen Pin schreiben, aber wenn der Pin ja extra für einen (1,5V oder Solar-) Motor sein soll wären 1 oder 2 Blöcke
MotorP12 an/aus vielleicht eine Idee. 
Vielleicht hab ich ja auch nur nicht den richtigen Block gefunden?

**Comment by HyperDuino** [comment]
For the regular MakerBit, the MOTOR pin is pin P10. 

![image](https://user-images.githubusercontent.com/13832091/52808923-ec7bf700-3043-11e9-9be1-8a75fb8fc74e.png)

The concern I would have is that we have motor options now as A, B and A+B. I suppose the label could be added of MOTOR but we should consider whether run motor MOTOR would confuse users of the MakerBit+R.

**Comment by MKleinSB** [comment]
Sorry that I wrote german, I thought Phillip is reading.
My Makerbit (V5.13) uses Pin 12 as Motorpin.

![wp_20190214_20_07_35_pro](https://user-images.githubusercontent.com/37311672/52811384-3c36de80-3095-11e9-8e52-4a78b864f8a0.jpg)

I´ just adapting the MakerBit to the Calliope Mini which will be used in german schools instead of the microbit and I designed an Adapter to use the MakerBit with the the Calliope Mini. Final Version will not be the red one from the picture above but will look like this:
![unbenannt](https://user-images.githubusercontent.com/37311672/52811648-e57dd480-3095-11e9-863c-6a1c67619ba5.png)
I will write a hackster projekt https://www.hackster.io/calliope-makerbit-team/makerbit-am-calliope-mini-wip-4ce1e2 to describe the possibilities of the MakerBit for german schools. So it would be nice to know the right number ;-)
Before I forget: One essential part of the MakerBit is the Media Linker (Firmata). Will you be able to compile it for the Calliope Mini? I provide the mapped pins of the Calliope Mini to you when you are able to do that,
that´s it for now and thanks!

Ah, forget the motor-problem, I will provide it in the hackster-project!

**Comment by HyperDuino** [comment]
Sorry, yes you are right! I had forgotten that we moved the motor control pin in v5.13 to P12.  v5.12 is P12, and v5.10 & 5.11 was P10.  This was a result of iterations to find the optimum overall pin assignments with the micro:bit. 

Are you saying that the Calliope Mini and/or Callio:bit can operate as a replacement for the micro:bit, and not need the micro:bit at all?

**Comment by MKleinSB** [comment]
Yes, you are right! I adapted all Makerbitextensions for the Calliope Mini. We still use V0 core at makecode.calliope.cc and the Calliope Mini has more Components (RGB-LED, Motordriver ..) so we´re missing one Analog Pin and the Pins are used different to the microbit, but with the callio:bit I solved it. Phillip will receive another Prototypeboard to test the motordriver on his +R board. The callio:bit(M) will be free available at upverter and i will provide free Gerberfile for PCB-Production.

**Comment by MKleinSB** [comment]
PCBs and Gerberfile ready and available at PCBWay 
[https://www.pcbway.com/project/shareproject/Calliobit_M_______Calliope_Mini_To_MakerBit_Bridge.html](https://www.pcbway.com/project/shareproject/Calliobit_M_______Calliope_Mini_To_MakerBit_Bridge.html)
Hackster project (not yet finished) at [https://www.hackster.io/calliope-makerbit-team/makerbit-am-calliope-mini-4ce1e2](https://www.hackster.io/calliope-makerbit-team/makerbit-am-calliope-mini-4ce1e2)
![img_7346](https://user-images.githubusercontent.com/37311672/53761894-f5cfe500-3ec6-11e9-96e5-a99e4fdab0d0.JPG)
Should I send you some?

**Comment by HyperDuino** [comment]
What is the difference in usage between the left and right pcbs? (female/male headers)

**Comment by MKleinSB** [comment]
The pcb is the same. Only in Calliopeblue and makerbitwhite. But some people already have soldered a male/female header back or in front of the 26pin expansionport of their Calliope Mini. In my other extensions I prefered male in Calliope and female in the extension. After I few month of use in my school I prefere female in the Calliope Mini so it´s more difficult to shortcircuit or break the headers. But the user have the choice. 
I hope it´s ok that I use your MakerBit Photos at the hackster Projekt. I only translated it to german.
PS: I think blue looks better ;-)
